### DISCUSSION TITLE
Migrate Collections

### DISCUSSION THREAD

**Discussion by GeoXydias** [outline/outline#4731] [Self Hosting] [answered]
Hi,

I manage to deploy `outline` on kubernetes. 
I was wandering if there is an easy way to migrate the collections from the existing outline instance to the one I deployed.

**Comment by tommoor** [comment, marked as answer]
You should use the export option on the old installation and the import option on the new installation, the two are compatible although this is a lossy process currently depending on the software version it should bring across all text and image content.

**Reply by GeoXydias** [reply]
Thanks for your respose @tommoor, 

I want to do it through API,
what I have in my mind is that I am doing a GET request to fetch the collections and then
a POST request in order to upload the collections to the new installation.  

Do you think this is something doable?

**Reply by tommoor** [reply]
Absolutely, anything the app can do you can do through the API.
